crates/astria-sequencer/src/sequencer.rs

```diff
@@ -159,6 +199,7 @@ fn start_grpc_server(
     storage: &cnidarium::Storage,
     mempool: Mempool,
     grpc_addr: std::net::SocketAddr,
+    optimistic_block_channels: Option<OptimisticBlockChannels>,
     shutdown_rx: oneshot::Receiver<()>,
 ) -> JoinHandle<Result<(), tonic::transport::Error>> {
```

Comment:
Relevant to some observations I made earlier: if `start_grpc_server` returned a tuple `(JoinHandle, GrpcServerChannel)` (terrible names, just for the sake of the argument), then other parts of the system could use `GrpcServerChannel` to communicate with its components without needing to be aware of what's going on inside.